README.md

```diff
@@ -324,6 +324,19 @@ Environment=TARANTOOL_PID_FILE=/var/run/tarantool/${app_name}.{instance_name}.pi
 Environment=TARANTOOL_CONSOLE_SOCK=/var/run/tarantool/${app_name}.{instance_name}.control
 ```
 
+#### Managing dymanic `box.cfg` parameters without restart
+
+Some `box.cfg` options are dynamic, it mean that it can be changed without
+instance restarting.
+See [parameters](https://www.tarantool.io/en/doc/latest/reference/configuration/#configuration-parameters)
+with "Dynamic: yes".
+
+Role changes this parameters without restarting the instance.
+If other parameters were changed, instance is restarted anyway.
```

Comment:
`were` -> `are`.

Otherwise you need to correct your mixed conditional (https://engblog.ru/mixed-conditionals)